AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -73,7 +66,7 @@ struct AuthenticationApp: App {
 //        // The scheme of the redirect URL is also specified in the Info.plist file.
 //        redirectURL: URL(string: "authexample://auth")!
 //    )
-//}
```

Comment:
Uncommenting the extension but leaving the property within commented prevents us from violating the swiftlint comment_spacing rule.